### DISCUSSION TITLE
error typegoose auto increment

### DISCUSSION THREAD

**Discussion by MelHern** [typegoose/auto-increment#26] [Q&A]
when in my model use:

@plugin(AutoIncrementSimple, [{ field: 'id' }])

error: cannot find name 'AutoIncrementSimple'

I installed the package like the instructions...

**Comment by hasezoey** [comment]
> error: cannot find name 'AutoIncrementSimple'

you have to import the function, this package provides 2 plugin functions: [`AutoIncrementSimple`](https://github.com/typegoose/auto-increment?tab=readme-ov-file#simple) and [`AutoIncrementID`](https://github.com/typegoose/auto-increment?tab=readme-ov-file#for-identification); this package does not have a default export
